CHANGELOG.md

```diff
@@ -2,9 +2,19 @@
 
 ### 2.x.x (In progress)
 
-***SDK***
+**SDK**
+
+- **Fixed:** support of `playsinline` instruction of  `Video` html tag
+
+- **Feature:** Added separate public `MiniAppUserInfoProtocol` to communicate with the host app for User profile related retrieval.
+- **Feature:** Added interfaces such as [getUserName](USERGUIDE.md#cuser-profile-details-username) and [getProfilePhoto](USERGUIDE.md#cuser-profile-details-profilephoto) in `MiniAppUserInfoProtocol` to retrieve username and profile photo respectively. Interfaces will be called only if user has agreed to the respective custom permission. i.e `rakuten.miniapp.user.USER_NAME` for [getUserName](USERGUIDE.md#cuser-profile-details-username) and `rakuten.miniapp.user.PROFILE_PHOTO` for [getProfilePhoto](USERGUIDE.md#cuser-profile-details-profilephoto)
+- **Feature:** Added support for Javascript bridge interface for User Info detail retrieval from Mini app.
+`getUserName()` and `geProfilePhoto()`
+- **Feature:** Updated public interface with `MiniAppUserInfoProtocol` support `MiniApp#create(appId:completionHandler:messageInterface:userInfoProtocol)`
```

Comment:
I think this seems too detailed for the change log. All four of these items are the same feature: providing user info to the mini app. I think this should be simplified and combined into a single change log entry which just mentions that we've added a new feature for providing user info, so they can implement `MiniAppUserInfoProtocol`, `getUsername`, and `getProfilePhoto`.

Reply:
Done